### DISCUSSION TITLE
Phone Camera

### DISCUSSION THREAD

**Discussion by rafikApp** [Budibase/budibase#6233] [General]
Hello everyone.
Can we use phone camera in the app?

**Comment by melohagan** [comment]
Hey @rafikApp 

The closest thing to this on the roadmap would be for QR scanning: https://github.com/Budibase/budibase/issues/4379

What would you like to use the phone camera for?

**Reply by rafikApp** [reply]
No not QR scanning, i would like to take normal pictures

**Reply by oswaldB** [reply]
For example I need to take photo to show car mechanics where is the problem.

**Comment by ChuckDobbs** [comment]
Being able to add a phone photo to a record really needs to be a basic feature. (Local or S3 storage doesn't matter). I have a number of use cases where this is essential. Inventory, Personnel, repair shops, sales trackers. Look at how the "AnyList" app works on an iPhone.
